### DISCUSSION TITLE
Aumentar tiempo de time out

### DISCUSSION THREAD

**Discussion by ElDukes** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#36] [Q&A]
Se puede aumentar el tiempo del time out? 

    "message": "Server error: `GET https://contpaqiapim.azure-api.net/comercial/api/requests/db2c2415-d00b-4f4b-fe0d-08dbbb82e8ad` resulted in a `500 Internal Server Error` response:\n<html><head><title>500 - The request timed out.</title></head><body>  <font color =\"#aa0000\">         <h2>500 - The requ (truncated...)\n",

**Comment by AndresRamos** [comment]
Se corrige en #38
